finality-aleph/src/party/mod.rs

```diff
@@ -196,6 +225,7 @@ where
         spawn_handle: spawn_handle.into(),
         phantom: PhantomData,
         unit_creation_delay,
+        authorities,
```

Comment:
Please add a comment on what is the intended content of this field and how it changes.

Reply:
Done